src/main/java/com/societegenerale/commons/plugin/service/InvokableRules.java

```diff
@@ -105,11 +131,11 @@ InvocationResult invokeOn(JavaClasses importedClasses) {
 
         InvocationResult result = new InvocationResult();
         for (Method method : archRuleMethods) {
-            checkForFailure(() -> invoke(method, instance, importedClasses))
+            checkForFailure(() -> invoke(method, null, importedClasses))
```

Comment:
ArchUnit examples use `static` fields and methods so no need for an instance here.

If it turns out an instance is needed, then more work is needed.